db/submission_test.go

```diff
@@ -59,3 +59,54 @@ func (this *DBTests) DBTestRemoveSubmission(test *testing.T) {
         }
     }
 }
+
+func (this *DBTests) DBTestFetchAttempts(test *testing.T) {
+    ResetForTesting();
+    defer ResetForTesting();
+
+    assignment := MustGetTestAssignment()
+
+    // User has submissions
```

Comment:
Also, this comment does not directly reflect the code below it.
In the context of this test, "grader@test.com" is really the "user".
We do not want to imply that "student@test.com" is the user.
(It actually tricked me for a second.)

So instead, say something like "Fetch attempts from another user."